covid19-dashboard/dc-covid19/src/setupTests.ts

```diff
@@ -1,3 +1,19 @@
+/**
+ Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
 // jest-dom adds custom jest matchers for asserting on DOM nodes.
```

Comment:
uncomment?

Reply:
This was commented by default by React. It was created using create-react-app